lib/kitchen/directions/bake_notes/bake_numbered_notes.rb

```diff
@@ -7,6 +7,9 @@ def self.v1(book:, classes:)
         classes.each do |c|
           book.chapters.notes("$.#{c}").each do |note|
             bake_note(note: note)
+            next unless note.exercises.count.positive?
```

Comment:
I think `.positive?` returns `true` on 0

Reply:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/26280712/114224819-30729a00-9937-11eb-913d-d953174daf89.png">
had to double-check -- it is `false` on 0!

Reply:
ok awesome! lol

Reply:
Kudos to y'all for checking the behavior of `positive?` 👏 

This could instead be 

```ruby
next unless note.exercises.any?
``` 

or

```ruby
next if note.exercises.none?
```

... but I think it is cleaner if this check is inside `bake_note_exercise`.  That code already has conditionals to return if there isn't a solution, so it seems reasonable to me that it could also be the place that bails out if there is no exercise.  

```ruby
def self.bake_note_exercise(note:)
  exercise = note.exercises.first
  return unless exercise
```